Backpack/Calendar/Classes/BPKCalendarColor.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+/**
+ * `BPKCalendarColor` contains a number of pre-defined colours commonly used when presenting a calendar.
+ */
+@interface BPKCalendarColor : NSObject
+
+/**
+ * The normal calendar background color.
+ */
+@property(class, readonly) UIColor *normalColor;
```

Comment:
Should we create a `BPKColorPair` class to avoid having two properties per class?

Reply:
Maybe, but the hard thing is the background color in one use case is the forground color in another 🤔

Reply:
Yeah you're right. I am thinking of the case when we have a pair were the `foregroundColor` is suitable to put on top of a view with the specific backgroundColor

Reply:
tbh I think it's probably yagni. We're gonna be the main consumers of pairs of colours anyway 🤷